content/tutorials/getting-started.ja.markdown

```diff
@@ -0,0 +1,101 @@
+---
+タイトル: はじめに
+カテゴリ: 全般
+難易度: かんたん
```

Comment:
This is the colored difficulty tag next to the title, and the value must be in English. Either "easy", "medium", or "hard".